### ISSUE TITLE
Word wrapping fails when end of line is a HTML tag

### ISSUE THREAD

**Issue by francisrath** [beautify-web/js-beautify#1055]
# Description
Formatting HTML with `wrap lines` does not break correctly when the last part of the broken line is a HTML-tag. If you replace `</strong>` with `</strong>x` it will break as expected.

# Input
The code looked like this before beautification:
```
<div class="aside-container">
    <div>
        <p>Alkohol forbrennes med en hastighet på omtrent 0,15‰ per time,
            men varierer fra person til person. Det er <strong>umulig</strong>
            å øke forbrenningen med vanlige råd som fysisk aktivitet eller
            ved å drikke mye vann. Det eneste som hjelper for å få alkohol
            ut av kroppen er å vente.</p>
    </div>
    <aside class="warning">
      ...
    </aside>
</div>
```

# Expected Output
The code should have looked like this after beautification:
```
<div class="aside-container">
    <div>
        <p>Alkohol forbrennes med en hastighet på omtrent 0,15‰ per time,
            men varierer fra person til person. Det er <strong>umulig</strong>
            å øke forbrenningen med vanlige råd som fysisk aktivitet eller
            ved å drikke mye vann. Det eneste som hjelper for å få alkohol
            ut av kroppen er å vente.</p>
    </div>
    <aside class="warning">
      ...
    </aside>
</div>
```

# Actual Output
The  code actually looked like this after beautification:
```
<div class="aside-container">
    <div>
        <p>Alkohol forbrennes med en hastighet på omtrent 0,15‰ per time,
            men varierer fra person til person. Det er <strong>umulig</strong>            å øke forbrenningen med vanlige råd som fysisk aktivitet eller
            ved å drikke mye vann. Det eneste som hjelper for å få alkohol
            ut av kroppen er å vente.</p>
    </div>
    <aside class="warning">
        ...
    </aside>
</div>
```

# Steps to Reproduce

## Environment
OS: Visual Studio Code 1.7.1 on Windows 10 and on jsbeautifier.org

## Settings
Example:
```json
{
    "indent_size": "4",
    "indent_char": " ",
    "max_preserve_newlines": "2",
    "preserve_newlines": true,
    "keep_array_indentation": false,
    "break_chained_methods": false,
    "indent_scripts": "normal",
    "brace_style": "collapse",
    "space_before_conditional": true,
    "unescape_strings": true,
    "jslint_happy": false,
    "end_with_newline": false,
    "wrap_line_length": "70",
    "indent_inner_html": false,
    "comma_first": false,
    "e4x": false
}
```

**Comment by francisrath** [comment]
Any updates on this?

**Comment by bitwiseman** [comment]
@francisrath 
This task needs someone to write tests for it and implement it.

**Comment by bitwiseman** [comment]
This particular issue is fixed in 1.8.0-rc6.  This output is still odd, but not this issue.
